docs/build/apps/dapp-frontend.mdx

```diff
@@ -19,16 +19,13 @@ Let's get started.
 
 You're going to need [Node.js](https://nodejs.org/en/download/package-manager/) v18.14.1 or greater. If you haven't yet, install it now.
 
-We want to initialize our current project as an Astro project. To do this, we can again turn to the `stellar contract init` command, which has a `--frontend-template` flag that allows us to pass the url of a frontend template repository. As we learned in [Storing Data](../smart-contracts/getting-started/storing-data.mdx#adding-the-increment-contract), `stellar contract init` will not overwrite existing files, and is safe to use to add to an existing project.
-
-From our `soroban-hello-world` directory, run the following command to add the Astro template files.
+We want to initialize our current project as an Astro project. From our `soroban-hello-world` directory, run the following command to add the Astro template files.
 
 ```sh
-stellar contract init ./ \
-  --frontend-template https://github.com/stellar/soroban-astro-template
+git clone git@github.com:stellar/soroban-template-astro.git
```

Comment:
this will clone the template repo's files into a new `soroban-template-astro` directory, inside `soroban-hello-world`. in order to use these template files, i think we'd need to instruct the user to copy them into the root of the `soroban-hello-world` project.

Reply:
Have something different here from Chad's PR